### PR TITLE
mysql support

### PR DESCRIPTION
**Pull request by ohadisraeli**

### CODE REVIEW

File: dataload/financial_data.py [hunk 1]
```diff
@@ -10,17 +10,17 @@
...
     port=config.REDIS_PORT,
     decode_responses=True
-
 )
 
 ELEMENT_LIST = [
     # general
-    'us-gaap:CommonStockSharesOutstanding'
+    # 'us-gaap:CommonStockSharesOutstanding',
+    'dei:EntityCommonStockSharesOutstanding',
```
Comment (otnielvh) [inline dataload/financial_data.py:18]: Is there a difference between upper and lower case? Maybe we can keep everything camel case, it makes it much easier to read

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    container_name: fin_mysql
+    image: mysql
+    ports:
+     - "3306:3306"
+    environment:
+       - MYSQL_ROOT_PASSWORD=mysql
+       - MYSQL_DATABASE=findb
+    volumes:
+       - $PWD/data/mysql/data:/var/lib/mysql
+       - $PWD/data/mysql/init:/docker-entrypoint-initdb.d/:ro
```
Comment (otnielvh) [inline docker-compose.yaml:21]: What's the last colon (:) for, is it a typo?

File: loaddata.py [hunk 3]
```diff
@@ -60,35 +67,38 @@ def _fetch_company_data(ticker: str, year: int, txt_url: str) -> None:
...
     Args:
         year int: The year to build the index for
         quarter int: The quarter to build the index between 1-4
     Returns:
-        List
+        None
     """
-    filing = '10-K'
-    exclude = '10-K/A'
+    filing = '|10-K|'
```
Comment (otnielvh) [inline loaddata.py:84]: It's kind of weird the filing has `|`

**Review by otnielvh — APPROVED** [review]

File: dataload/financial_data.py (hunk 1 above)
Comment (ohadisraeli) [reply dataload/financial_data.py:18]: This must be aligned with the tags I think... we can try and see if it works. But I think that in this format it is more clear anyway.

File: docker-compose.yaml (hunk 2 above)
Comment (ohadisraeli) [reply docker-compose.yaml:21]: Which line? Do not see one without a purpose

File: loaddata.py (hunk 3 above)
Comment (ohadisraeli) [reply loaddata.py:84]: Then there is no need to exclude, there are more cases then 10-K/A

File: docker-compose.yaml (hunk 2 above)
Comment (otnielvh) [reply docker-compose.yaml:21]: `:ro`

File: data/mysql/init/init.sql [hunk 4]
```diff
@@ -0,0 +1,11 @@
+CREATE USER 'finuser'@'%' IDENTIFIED WITH mysql_native_password BY 'fin';
+GRANT ALL PRIVILEGES ON * . * TO 'finuser'@'%';
+
+CREATE TABLE IF NOT EXISTS sec_idx (
+    cik INT,
+    year INT,
+    company TEXT,
+    report_type TEXT,
+    url TEXT,
+    PRIMARY KEY (sec, year)
```
Comment (otnielvh) [inline data/mysql/init/init.sql:10]: Should be `PRIMARY KEY (cik, year)`

**Pull request merged**